Sources/Hummingbird/HTTP/MediaType+FileExtension.swift

```diff
@@ -0,0 +1,97 @@
+import HummingbirdCore
+
+extension MediaType {
+    public struct FileExtension: RawRepresentable, ExpressibleByStringLiteral, Sendable, Hashable {
```

Comment:
Public types need documentation as do their public members, methods

Reply:
I'll leave resolving conversations to you to make sure the changes are to your specifications.